Create a template for all dimensions (#6560)
netdata/netdata: health/README.md

     have spaces in their names). This accepts Netdata simple patterns and the `match-ids` and
     `match-names` options affect the searches for dimensions.

> - `foreach DIMENSIONS` is optional and it will always be the last parameter. The rules for dimension here follow the same

Reviewer: These comments are very confusing. Please get help from @joelhans or @prhomhyse to improve the content here.

Reviewer 2: Here is my recommendation for altering the grammar/structure of this paragraph. This one sentence—`It is used to set the group of the dimensions of a chart that will have the same rule to raise an alarm.`—is difficult for me to understand and I may have rewritten it wrong. Let's make sure it's right before any changes are made.

> `foreach DIMENSIONS` is optional, will always be the last parameter, and uses the same `,`/`|` rules as the `of` parameter. Each dimension you specify in `foreach` will use the same rule to trigger an alarm. If you set both `of` and `foreach`, Netdata will ignore the `of` parameter and replace it with one of the dimensions you gave to `foreach`.

I'm having trouble understanding how I would try to format this line if I were going to use each of these parameters. @thiagoftsm maybe you could write an example template using `lookup` and `foreach` and we could document what each parameter accomplishes? We could add this to the **Examples** section, perhaps. It may also just help me better understand how to best document this rule.

Author: Hi @joelhans ,

I copied and paste your suggestion in the documentation, I also created two examples in the section cited, in the examples I decided to focus in the `lookup` line that is the greatest difference from the others, but case you find that is good to describe each line I can do.
Thank you to help me here!

When Netdata detects a dropped packet, it will add the chart and it will automatically attach this
alarm to it.

### Example 5

Check if user or system dimension is using more than 50% of cpu:

```
 alarm: dim_template
    on: system.cpu
    os: linux
lookup: average -3s percentage foreach system,user
 units: %
 every: 10s
  warn: $this > 50
  crit: $this > 80
```

The `lookup` line will calculate the average CPU usage from system and user in the last 3 seconds, due to the fact we have

> the foreach in the `lookup` line, Netdata will create two independent alarms called respectively `dim_template_system`

Author: Here I imported the Portuguese to English, thank you very much!

and `dim_template_user` that will have all the other parameters shared among them.

### Example 6

> Check if all dimensions is using more than 50% of cpu:

Author: I copied and paste, and I forgot to change, thanks!

```
 alarm: dim_template
    on: system.cpu
    os: linux
lookup: average -3s percentage foreach *
 units: %
 every: 10s
  warn: $this > 50
  crit: $this > 80
```

The `lookup` line will calculate the average of CPU usage from system and user in the last 3 seconds, here we will

> have the same behavior of the previous example, we will create alarms for all dimensions of the chart.

Reviewer 2: ```suggestion
In this case, Netdata will create alarms for all dimensions of the chart.
```

Author: I fixed all the grammar according your suggestions.

When Netdata detects a dropped packet, it will add the chart and it will automatically attach this
alarm to it.

### Example 5

Check if user or system dimension is using more than 50% of cpu:

```
 alarm: dim_template
    on: system.cpu
    os: linux
lookup: average -3s percentage foreach system,user
 units: %

>  every: 1s

Reviewer 3: minimum recommended `every` is 10s, check other exanmples

Author: Thank you @ilyam8 , I fixed in both examples.

  warn: $this > 50
  crit: $this > 80
```

> The `lookup` line will calculate the average of CPU usage from system and user in the last 3 seconds, due the fact we have

Reviewer 4: The `lookup` line will calculate the average of CPU usage from system and user in the last 3 seconds, due to the fact we have

Author: Thank you @prhomhyse , I overwrite the whole text with your suggestion.

    os: linux
lookup: average -3s percentage foreach system,user
 units: %
 every: 10s
  warn: $this > 50
  crit: $this > 80
```

> The lookup line will calculate the average CPU usage from system and user in the last 3 seconds, due to the fact we have

Reviewer 4: **lookup** should be an inline code `lookup`:

Add backticks to it, it will look like the previous commit.

Author: When I pasted your correction I lost it, thank you for the review, I brought it back now.
